### DISCUSSION TITLE
Add custom class to grid column

### DISCUSSION THREAD

**Discussion by ArtMin96** [filamentphp/filament#8388] [Help]
How can I add a class here so I can style it?
![image](https://github.com/filamentphp/filament/assets/29732308/338c2571-f715-48a3-bc9b-6e400bc47577)

**Comment by leandrocfe** [comment]
maybe via `->extraAttributes(['class' => 'xx'])` ?

**Reply by ArtMin96** [reply]
It will affect the child element. That's how I added the "address-fields" class, which you can see in the screenshot. I solved the problem with another option, I just left it open because somebody might recommend a better option.
Thanks as always

**Reply by dissto** [reply]
How did you solve it?

**Reply by ArtMin96** [reply]
@dissto `.col-\[--col-span-default\]:has(.your-class) {}`
